components/card/src/vwc-card.ts

```diff
@@ -46,13 +46,13 @@ export class VWCCard extends LitElement {
 
 	@property({
 		reflect: true,
-		attribute: 'supporting-text',
+		attribute: 'card-text',
```

Comment:
Just `text` maybe?

Reply:
`Description`?

Reply:
only text seemed a bit short...
Not sure that this is "Description"

Reply:
it's not aligned with convention. otherwise it is all should be cardHeaderIcon, cardTitle, cardSub... redundant if you ask me

Reply:
changed to _text_